src/applications/disability-benefits/all-claims/content/disabilityLabels.js

```diff
@@ -0,0 +1,837 @@
+  620: 'back sprain',
+  630: 'back strain',
+  640: 'basaloma',
+  650: 'Bell\'s palsy',
+  660: 'beriberi (including beriberi heart disease)',
+  670: 'bipolar disorder',
+  680: 'Birth Defect in Child',
+  690: 'blackwater fever',
+  700: 'bladder condition',
+  710: 'bladder stone(s)',
+  720: 'blindness (including light perception only)',
+  730: 'blood clot(s)',
+  740: 'Boeck\'s disease',
+  750: 'bone atrophy',
+  760: 'bone condition',
+  770: 'bone fracture(s)',
+  780: 'bowel condition',
+  790: 'brain hemorrhage',
+  800: 'brain thrombosis',
+  810: 'brain tumor(s)',
+  820: 'breast condition, bilateral',
+  830: 'breast condition, left',
+  840: 'breast condition, right',
+  850: 'bronchiectasis',
+  860: 'bronchitis',
+  870: 'bruise',
+  880: 'Buerger\'s disease',
+  890: 'bulbar palsy',
+  900: 'burn(s)',
+  910: 'bursitis',
+  920: 'CAD',
+  930: 'Caisson disease',
+  940: 'calculus - bladder',
+  950: 'calculus - gallbladder',
+  960: 'calculus - kidney',
+  970: 'cancer',
+  980: 'cancer - bile ducts',
+  990: 'cancer - bone',
+  1000: 'cancer - brain',
+  1010: 'cancer - breast',
+  1020: 'cancer - bronchus',
+  1030: 'cancer - cheek',
+  1040: 'cancer - colon',
+  1050: 'cancer - esophagus',
+  1060: 'cancer - gall bladder',
+  1070: 'cancer - gastrointestinal tract',
+  1080: 'cancer - gum',
+  1090: 'cancer - kidney',
+  1100: 'cancer - larynx',
+  1110: 'cancer - lip',
+  1120: 'cancer - liver',
+  1130: 'cancer - lung',
+  1140: 'cancer - lymph nodes',
+  1150: 'cancer - nasopharyngeal',
+  1160: 'cancer - ovarian',
+  1170: 'cancer - pancreas',
+  1180: 'cancer - pharyngolaryngeal',
+  1190: 'cancer - pharynx',
+  1200: 'cancer - prostate',
+  1210: 'cancer - rectum',
+  1220: 'cancer - renal pelves',
+  1230: 'Cancer - Respiratory',
+  1240: 'cancer - salivary gland',
+  1250: 'Cancer - Skin',
+  1260: 'cancer - small intestine',
+  1270: 'cancer - squamous cell, of the skin',
+  1280: 'cancer - stomach',
+  1290: 'cancer - testicle',
+  1300: 'cancer - thyroid',
+  1310: 'cancer - tongue',
+  1320: 'cancer - trachea',
+  1330: 'cancer - ureters',
+  1340: 'cancer - urethra',
+  1350: 'cancer - urinary bladder',
+  1360: 'cancer - urinary tract',
+  1370: 'cancer - urogenital system (except prostate)',
+  1380: 'carate (AKA pinta)',
+  1390: 'carcinoma',
+  1400: 'cardiovascular disease',
+  1410: 'cardiovascular signs or symptoms',
+  1420: 'carpal tunnel syndrome',
+  1430: 'cartilage',
+  1440: 'cataract',
+  1450: 'cataracts, posterior subcapsular',
+  1460: 'catarrhal fever (common cold)',
+  1470: 'cavity',
+  1480: 'central nervous system tumors',
+  1490: 'cerebral accident',
+  1500: 'cerebral hemorrhage',
+  1510: 'cerebral vascular accident',
+  1520: 'cerebrovascular accident',
+  1530: 'cerebrovascular hemorrhage',
+  1540: 'cervical sprain',
+  1550: 'cervical strain',
+  1560: 'cervicitis',
+  1570: 'CHF',
+  1580: 'chest pain',
+  1590: 'Chloracne or other acneform disease consistent w/ chloracne',
```

Comment:
Should that first `Chloracne` be capitalized? Also, the `w/` abbreviation is inconsistent, I think.

Not sure how much it matters, but while we're here... :man_shrugging: